.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,17 @@
+name: reviewdog
+
+on:
+  pull_request:
+
+jobs:
+  golangci-lint:
+    name: lint checks by using golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+
+      - name: golangci-lint
+        uses: reviewdog/action-golangci-lint@v2
+        with:
+          golangci_lint_version: v1.46.2
```

Comment:
https://github.com/golangci/golangci-lint/releases/tag/v1.46.2
(the current latest version)